packages/proto-loader/src/index.ts

```diff
@@ -395,6 +395,23 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export function fromJSON(
+  json: Protobuf.INamespace,
+  root?: Protobuf.Root
+): PackageDefinition {
+  const options: Options = json.options || {};
+  const newRoot: Protobuf.Root = root || new Protobuf.Root();
+  if (!!options.includeDirs) {
```

Comment:
My understanding is that we are not loading any files, so `includeDirs` is not relevant.